sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -755,9 +755,12 @@ message FunctionSpec {
   // passed as-is.
   string urn = 1;
 
+  // (Deprecated)
```

Comment:
If we are changing the name of this field, might as well remove it rather than deprecating?